src/main/resources/docs/multiple-tests/with-config-file/patterns.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<module name="root">
+    <!-- To ignore config files from analysis -->
+    <module name="BeforeExecutionExclusionFileFilter">
+        <!-- value can be a regex matching files to ignore -->
+        <property name="fileNamePattern" value=".codenarcrc"/>
```

Comment:
Here `value` is a regex, so you should escape the dot.
like `\.codenarcrc`

Reply:
Done